ng2/5-piratenameservice/lib/src/name_service.dart

```diff
@@ -12,16 +12,31 @@ class NameService {
   static final Random _indexGen = new Random();
 
   final _names = <String>[
-    'Anne', 'Mary', 'Jack', 'Morgan', 'Roger',
-    'Bill', 'Ragnar', 'Ed', 'John', 'Jane'
+    'Anne',
+    'Mary',
+    'Jack',
+    'Morgan',
+    'Roger',
+    'Bill',
+    'Ragnar',
+    'Ed',
+    'John',
+    'Jane',
   ];
   final _appellations = <String>[
-    'Jackal', 'King', 'Red', 'Stalwart', 'Axe',
-    'Young', 'Brave', 'Eager', 'Wily', 'Zesty'
+    'Jackal',
+    'King',
+    'Red',
+    'Stalwart',
+    'Axe',
+    'Young',
+    'Brave',
+    'Eager',
+    'Wily',
+    'Zesty',
   ];
 
-  String _randomFirstName() =>
-      _names[_indexGen.nextInt(_names.length)];
+  String _randomFirstName() => _names[_indexGen.nextInt(_names.length)];
```

Comment:
Since the code doesn't appear in the page, I'm fine with dartfmt defaults.

However, I'd like some sort of solution for this problem of making code snippets legible without scrolling. Scrolling is better than wrapping, but they're both unfriendly to readers.